mavros_msgs/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ add_message_files(
   OpticalFlowRad.msg
   OverrideRCIn.msg
   ParamValue.msg
+  ManualControls.msg
```

Comment:
Why plural form? Please name it `ManualControl.msg` like corresponding mavlink message.